src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,65 @@
+package core.basesyntax.model;
+
+import java.util.NoSuchElementException;

```

Comment:
This import seems unused in the current context of the class `FruitTransaction`. Make sure to remove any unused imports to keep the code clean.